src/framework/control.ml

```diff
@@ -706,7 +706,8 @@ let compute_cfg file =
 
 (** The main function to perform the selected analyses. *)
 let analyze change_info (file: file) fs =
-  if (get_bool "dbg.verbose") then print_endline "Generating the control flow graph.";
-  let (module CFG) = compute_cfg file in
-  MyCFG.current_cfg := (module CFG);
-  analyze_loop (module CFG) file fs change_info
+  GobConfig.with_immutable_conf @@ fun () ->
+    if (get_bool "dbg.verbose") then print_endline "Generating the control flow graph.";
+    let (module CFG) = compute_cfg file in
+    MyCFG.current_cfg := (module CFG);
+    analyze_loop (module CFG) file fs change_info
```

Comment:
I think this should be in `analyze_loop` around `A.analyze` instead because that restarting modifies options for the restarted run.